cli/sawtooth_cli/rest_client.py

```diff
@@ -95,7 +95,7 @@ def _get(self, path, **queries):
         if code == 200:
             return json_result
         elif code == 404:
-            return None
+            raise CliException("404 not found: {} ".format(path))
```

Comment:
I would prefer a little more informative error message here. Something like: `"There is no resource at the requested address: {}"`. The fact that the HTTP designation happens to be `404 Not Found` seems utterly unimportant to me.